src/fractal_faim_hcs/MetaSeriesUtils_dask.py

```diff
@@ -0,0 +1,302 @@
+    (nx_tot, ny_tot) = sample_fused.shape
+    img_dtype = sample_fused.dtype
+
+    # create dask array of assembled planes
+    fns_da = da.from_array(
+        fns_np,
+        chunks=((fns_np.shape[0],)+(1,)*len(fns_np.shape[1:]))
+    )
+    imgs_fused_da = da.map_blocks(
+        _fuse_xy,
+        fns_da,
+        chunks=da.core.normalize_chunks(
+            (1,) * len(fns_da.shape[1:]) + (nx_tot, ny_tot),
+            shape=fns_da.shape[1:] + (nx_tot, ny_tot),
+        ),
+        drop_axis=0,
+        new_axis=range(len(fns_da.shape) - 1, len(fns_da.shape) + 1),
+        meta=np.asanyarray([]).astype(sample_fused.dtype),
+    
+        img_shape=(nx_tot, ny_tot),
+        img_dtype=img_dtype
+    )
+
+
+    # GENERATE ROI_TABLES AND PX_METADATA:
+    # (by only loading one plane)
+    # TODO: Could also be done without loading images of entire plane
+    fns_sel = fns_da[:, channels_full_planes[0], 0].compute()
+    files = pd.DataFrame(fns_sel, columns=["path"])
+    px_metadata, _, _, _, roi_tables = get_img_YX(assemble_fn, files)
+
+    px_metadata["z-scaling"] = z_sampling
+
+    for roi_table in roi_tables.values():
+        roi_table["len_z_micrometer"] = z_sampling * (imgs_fused_da.shape[1] - 1)
+
+
+    # LOAD CHANNEL-METADATA
+    def load_channel_metadata(fn):
+        ms_metadata = load_metaseries_tiff_metadata(fn)
+        return _build_ch_metadata(ms_metadata)
+    
+    channel_metadata = []
+    for c in range(fns_np.shape[1]):
+        if (c in channels_full_planes) or (c in channels_one_plane):
+            for z in range(fns_np.shape[2]):
+                if fns_np[0,c,z] != '':
+                    channel_metadata.append(
+                        load_channel_metadata(fns_np[0,c,z])
+                    )
+                    break
+        else:
+            channel_metadata.append(
+                {
+                    "channel-name": "empty",
+                    "display-color": "000000",
+                }
+            )
+
+    # LOAD UINTHISTOGRAMS
+    # TODO: See if this can be done more efficiently
+    def _load_histo(x):
+        histo = UIntHistogram(x)
+        return np.array([[histo]])
+
+    histos_da = da.map_blocks(
+        _load_histo,
+        imgs_fused_da,
+        chunks=(1, 1),
+        drop_axis=(
+            2,
+            3,
+        ),
+        meta=np.asanyarray([UIntHistogram()]),
+    )
+    histos = histos_da.compute()
+
+    channel_histograms = []
+    for c, _ in enumerate(channels):
+        channel_histogram = UIntHistogram()
+        for z, _ in enumerate(planes):
+            channel_histogram.combine(histos[c, z])
+        channel_histograms.append(channel_histogram)
+    for c in channels_empty:
+        channel_histograms[c] = UIntHistogram()
+
+
+    return (
+        imgs_fused_da,
+        channel_histograms,
+        channel_metadata,
+        px_metadata,
+        roi_tables,
+    )
+
+
+
+def load_metaseries_tiff_metadata(path: Path) -> dict:
```

Comment:
Is there a reason to have `load_metaseries_tiff_metadata` copied over and not use the one from https://github.com/fmi-faim/faim-hcs/pull/36
This approach would duplicate all the "Lumencor * Intensity" for ZMB data in the metadata.

Is it just the line `metadata["PixelType"] = str(data.dtype)`? Does everything else run much faster and that take a lot of time?